dev/app.js

```diff
@@ -122,6 +122,8 @@ search.addWidget(
 
 search.addWidget(
   instantsearch.widgets.refinementList({
+    collapsable: true,
+    collapsed: true,
```

Comment:
What about we change the name of the option to `initiallyCollapsed` or something that translate the fact that it is for the initial state of the widget?